.circleci/config.yml

```diff
@@ -57,6 +57,42 @@ jobs:
           name: Scan dependencies for vulnerabilities
           working_directory: ~/workbench/api
           command: ./project.rb gradle dependencyCheckAnalyze --info
+  api-build-integration-test:
+    docker:
+      - image: allofustest/workbench:buildimage-0.0.10
+    working_directory: ~/workbench
+    environment:
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb
+    steps:
+      - checkout
+      - run:
+          name: Fetch Submodules
+          command: git submodule update --init
+      - restore_cache:
+          keys:
+          - api-cache-{{ checksum "~/workbench/api/build.gradle" }}
+          - api-cache-
+      - run:
+          working_directory: ~/workbench
+          command: ci/activate_creds.sh api/circle-sa-key.json
```

Comment:
Ah, I think I finally figured this out. I think we're calling activate-creds.sh solely to get permission to call gsutil below. A comment here and/or above would be great if we can confirm this was the sole intent here (@dmohs ?)